docs/installation.md

```diff
@@ -295,27 +295,17 @@ If you use sbt to install a pre-release, be sure to add the following setting:
 resolvers += Resolver.sonatypeRepo("snapshots")
 ```
 
-### Native image
+### Native binaries
 
-For macOS and Linux, it's possible to download pre-built GraalVm native binaries
-with instant startup and fast performance for short-lived Scalafmt runs.
+It's possible to download pre-built Scala Native binaries
```

Comment:
This might be controversial, but I feel like fixing the `install-scalafmt-native.sh`  is a lot more work and scala cli already properly downloads binaries.

Reply:
And also on all platforms

Reply:
that sounds good to me, no duplication